Adding FPG Risk Effects Page (#1065)
ihmeuw/vivarium_research: docs/source/gbd2019_models/risk_effects/fpg/index.rst

:math:`\text{RR(x)} = {\text{RR}_0}^{\max\left((x-\text{TMREL}), 0\right)}`

Where RR(x) is the RR at exposure level x and RR\ :sub:`0`\  is the 
increase in RR for each 1 mmol/L above the TMREL. GBD used 
DisMod-MR 2.1 to pool effect sizes from included studies and generate 
a dose-response curve for each of the outcomes associated with high SBP. 
The tool enabled GBD to incorporate random effects across studies and 
include data with different age ranges. RRs were used universally for 
all countries and the meta-regression only helped to pool the three 
major sources and produce RRs with uncertainty and covariance across 
ages taking into account the uncertainty of the data points. 

Theoretical minimum-risk exposure level
+++++++++++++++++++++++++++++++++++++++

The theoretical minimum-risk exposure level (TMREL) for FPG is 4.8-5.4 mmol/L for those risk-outcome pairs where risk is assessed on a continuous basis. This was calculated by taking the person-year 
weighted average of the levels of FPG that were associated with the lowest risk of mortality in the pooled analyses of prospective cohort studies. The TMREL is no diabetes for those outcomes where risk 

> is assessed on a categorical basis. The risk-outcome pairs are listed below, along with whether they are continuous or categorical.  

Reviewer: Is this a range because it varies by age/sex? Or is the bottom end of the range a minimum value for FPG exposure? This wasn't clear to me on a quick read!

   * - Categorical
     - Liver cancer due to other causes
   * - Categorical
     - Pancreatic cancer
   * - Categorical
     - Ovarian cancer
   * - Categorical
     - Colorectal cancer
   * - Categorical
     - Bladder cancer
   * - Categorical
     - Lung cancer
   * - Categorical
     - Breast cancer
   * - Categorical
     - Glaucoma
   * - Categorical
     - Cataracts
   * - Categorical
     - Dementia

[GBD-2019-Capstone-Appendix-FPG2]_

.. list-table:: GBD 2019 Restrictions
   :widths: 15 15 20
   :header-rows: 1

   * - Restriction Type
     - Value
     - Notes
   * - Male only
     - False
     -
   * - Female only
     - False
     -
   * - YLD only
     - False
     -
   * - YLL only
     - False
     -
   * - Age group start
     - 10
     - [25, 29 years) 
   * - Age group end
     - 235
     - [95, 125 years) 

Risk Outcome Pair #1: Ischemic heart disease
++++++++++++++++++++++++++++++++++++++++++++

:ref:`See ischemic heart disease documentation <2019_cause_ihd>`

The relative risks apply to the incidence rates of acute 
myocardial infarction. These are arrows labeled 1 on the IHD cause diagram. They should be 
applied using the formula: 

> incidence(i) = incidence*(1-PAF\ :sub:`r105`\)*RR^{max((BMI_i - TMREL),0)} 

Reviewer 2: ```suggestion
incidence(i) = incidence*(1-PAF\ :sub:`r105`\)*RR^{max((FPG_i - TMREL),0)} 
```

The association was evaluated at the cause level, but the 
associations should be applied to the incidence rates for 
both nonfatal components of ischemic heart disease. The 
relative risk for GBD 2019 is for a 1-unit increase in FPG.

PAFs and relative risks can be pulled using the following code::

  rrs = get_draws(gbd_id_type='rei_id', gbd_id=105, source='rr', year_id=2019, gbd_round_id=6, status='best', decomp_step='step4') 

  pafs = get_draws(gbd_id_type=['rei_id', 'cause_id'], gbd_id=[105, 493], source='burdenator', measure_id=2, metric_id=2, year_id=2019, gbd_round_id=6, status='best', decomp_step='step5') 

Risk Outcome Pair #2: Ischemic stroke
+++++++++++++++++++++++++++++++++++++

:ref:`See ischemic stroke documentation <2019_cause_ischemic_stroke>`

The relative risks apply to the incidence rates of acute 
ischemic stroke. These are arrows 1 and 3 on in the ischemic 
stroke cause model. They should be applied using the formula: 

> incidence(i) = incidence*(1-PAF\ :sub:`r105`\)*RR^{max((BMI_i - TMREL),0)} 

Reviewer 2: ```suggestion
incidence(i) = incidence*(1-PAF\ :sub:`r105`\)*RR^{max((FPG_i - TMREL),0)} 
```

> The relative risk for GBD 2019 is for a 1-unit increase in BMI. 

Reviewer 2: ```suggestion
The relative risk for GBD 2019 is for a 1 mmol/L increase in FPG. 
```

(can you confirm this?)

Author: Yes! Thanks for the small catches

PAFs and relative risks can be pulled using the following code:: 

  rrs = get_draws(gbd_id_type='rei_id', gbd_id=105, source='rr', year_id=2019, gbd_round_id=6, status='best', decomp_step='step4') 

  pafs = get_draws(gbd_id_type=['rei_id', 'cause_id'], gbd_id=[105, 495], source='burdenator', measure_id=2, metric_id=2, year_id=2019, gbd_round_id=6, status='best', decomp_step='step5') 

Assumptions and Limitations
+++++++++++++++++++++++++++

The quantity of interest is exposure to the mean FPG level; we assume full reversibility of risk and do not account for duration of exposure to FPG values above the range of the TMREL. 

> We are not including diabetes as a cause in our model, which is a PAF of 

Reviewer 2: ```suggestion
We are not including diabetes as a cause in our model, which is a PAF-of-one
```

> 1 cause with FPG. This means that while FPG affects IHD and stroke, we will 

Reviewer 2: ```suggestion
cause with FPG. This means that while FPG affects IHD and stroke, we will 
```
